src/ast2ram/AstToRamTranslator.cpp

```diff
@@ -678,26 +495,12 @@ Own<ram::Sequence> AstToRamTranslator::generateProgram(const ast::TranslationUni
     return mk<ram::Sequence>(std::move(res));
 }
 
-void AstToRamTranslator::preprocessAstProgram(ast::TranslationUnit& tu) {
-    // Finalise polymorphic types in the AST
-    finaliseAstTypes(tu);
-
-    // Replace ADTs with record representatives
-    removeADTs(tu);
-}
-
 Own<ram::TranslationUnit> AstToRamTranslator::translateUnit(ast::TranslationUnit& tu) {
-    // Start timer
-    auto ram_start = std::chrono::high_resolution_clock::now();
-
     /* -- Set-up -- */
-    // Set up the translator
+    auto ram_start = std::chrono::high_resolution_clock::now();
```

Comment:
Is this used?

Reply:
Yep, in the debug report section later on.